packages/frameworks/src/frameworks.ts

```diff
@@ -1719,7 +1719,7 @@ export const frameworks = [
       },
       devCommand: {
         placeholder: 'vite',
-        value: 'vite --port $PORT',
+        value: 'vite --port $PORT --host 0.0.0.0',
```

Comment:
Is this backwards compatible with older versions of vite? When was this introduced?

Reply:
I think the correct solution here is to update `vercel dev` to support both ipv4 and ipv6 since we'll run into this problem more often as other frameworks update to ipv6.

Reply:
This change is backwards compatible and I link the changelog in the initial issue which shows when it was introduced.

Vite changed the default bind address from `0.0.0.0` to `127.0.0.1`, I don't think it's to do with Ipv4 and Ipv6. It might be more correct to fix this issue on the `vercel dev` side but I have zero familiarity with the Vercel codebase and just wanted to push this fix as a patch to help people using Vite and Vercel.

Reply:
This will solve it for all frameworks, not just vite: #8542